bot/exts/utilities/githubinfo.py

```diff
@@ -293,54 +311,28 @@ async def github_user_info(self, ctx: commands.Context, username: str) -> None:
 
         await ctx.send(embed=embed)
 
-    @github_group.command(name="repository", aliases=("repo",))
-    async def github_repo_info(self, ctx: commands.Context, *repo: str) -> None:
-        """
-        Fetches a repositories' GitHub information.
-
-        The repository should look like `user/reponame` or `user reponame`.
-        """
-        repo = "/".join(repo)
-        if repo.count("/") != 1:
-            embed = discord.Embed(
-                title=random.choice(NEGATIVE_REPLIES),
-                description="The repository should look like `user/reponame` or `user reponame`.",
-                colour=Colours.soft_red
-            )
-
-            await ctx.send(embed=embed)
-            return
-
-        async with ctx.typing():
-            repo_data, _ = await self.fetch_data(f"{GITHUB_API_URL}/repos/{quote(repo)}")
-
-            # There won't be a message key if this repo exists
-            if "message" in repo_data:
-                embed = discord.Embed(
-                    title=random.choice(NEGATIVE_REPLIES),
-                    description="The requested repository was not found.",
-                    colour=Colours.soft_red
-                )
-
-                await ctx.send(embed=embed)
-                return
+    @tasks.loop(hours=24)
+    async def refresh_repos(self) -> None:
+        """Refresh self.repos with latest PyDis repos."""
+        self.repos, _ = await self.fetch_data(REPOSITORY_ENDPOINT.format(org="python-discord"))
```

Comment:
If you store self.repos as a dictionary of repo.name to repo data you won't need to use a for loop below and can instead check `name in self.repos`

I'm also not sure what I think about making the repo command use this cache: it seems to solve a problem that didn't exist with no apparent gain, though I'm probably overlooking something.